lib/cisco_node_utils/bgp_af.rb

```diff
@@ -57,6 +57,7 @@ def self.afs
     end
 
     def create
+      feature_nv_overlay_enable unless feature_nv_overlay_enabled?
```

Comment:
This does not feel safe. You're enabling nv overlay for every bgp address-family regardless of whether they need this feature or the platform even supports it. Won't this raise an error on some platforms?